src/staking-v3/components/data/DataList.vue

```diff
@@ -143,6 +149,44 @@ export default defineComponent({
       () => tiersConfiguration.value.numberOfSlots - dAppTiers.value.dapps.length
     );
 
+    const dappsPerTier = computed<number[]>(() => {
```

Comment:
Move computed fields to `useLeaderboard`.

Reply:
Done.